test/validations.test.js

```diff
@@ -517,6 +517,51 @@ describe('validations', function() {
         })).should.be.false;
       });
     });
+
+    it('passes case insensitive validation', function(done) {
+      User.validatesUniquenessOf('email', {ignoreCase: true});
+      var u = new User({email: 'hey'});
+      Boolean(u.isValid(function(valid) {
+        valid.should.be.true;
+        u.save(function() {
+          var u2 = new User({email: 'HEY'});
+          u2.isValid(function(valid) {
+            valid.should.be.false;
+            done();
+          });
+        });
+      })).should.be.false;
+    });
+
+    it('passes case insensitive validation with string that needs escaping', function(done) {
+      User.validatesUniquenessOf('email', {ignoreCase: true});
+      var u = new User({email: 'me+me@my.com'});
+      Boolean(u.isValid(function(valid) {
+        valid.should.be.true;
```

Comment:
https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access

I believe `.true` is a function in recent versions of shouldjs.

```js
valid.should.be.true();
```